CMakeLists.txt

```diff
@@ -156,10 +156,35 @@ set(GMOCK_LIBS
 set(CMAKE_CXX_STANDARD 14)
 
 # Common configuration for all build modes.
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
```

Comment:
Huh, I'm surprised we didn't already have this, but sure, makes sense.

Reply:
I think this parameter is coming down from travis. Do we need to explicitly set it in cmake?